src/look_transform.rs

```diff
@@ -109,14 +107,11 @@ fn look_transform_system(
     mut cameras: Query<(&LookTransform, &mut Transform, Option<&mut Smoother>)>,
 ) {
     for (look_transform, mut scene_transform, smoother) in cameras.iter_mut() {
-        let effective_look_transform = if let Some(mut smoother) = smoother {
-            smoother.smooth_transform(look_transform)
-        } else {
-            *look_transform
+        match smoother {
+            Some(mut s) if s.enabled => {
+                *scene_transform = s.smooth_transform(look_transform).into()
+            }
+            _ => (),
```

Comment:
You could avoid this empty branch if you did:

```
if let Some(mut smoother) = smoother {
    if smoother.enabled {
        *scene_transform = s.smooth_transform(look_transform).into()
    }
}
```

Reply:
Personally I find it better that way but it's really a matter of taste so I won't mind if you prefer to change it ;)